app/config/FrontendAppConfig.scala

```diff
@@ -57,8 +57,10 @@ class FrontendAppConfig @Inject()(configuration: Configuration) {
 
   lazy val taxReliefPerWeek2019: Int = configuration.getOptional[Int]("taxRelief.2019.poundsPerWeek").getOrElse(4)
   lazy val taxReliefPerWeek2020: Int = configuration.getOptional[Int]("taxRelief.2020.poundsPerWeek").getOrElse(6)
-  lazy val taxReliefMaxPerYear2019: Int = configuration.getOptional[Int]("taxRelief.2019.maxPerYear").getOrElse(212)
+  lazy val taxReliefPerWeek2021: Int = configuration.getOptional[Int]("taxRelief.2021.poundsPerWeek").getOrElse(6)
+  lazy val taxReliefMaxPerYear2019: Int = configuration.getOptional[Int]("taxRelief.2019.maxPerYear").getOrElse(56)
```

Comment:
No changes needed, just showing workings out.

Number of weeks between 1/1/2020 and 5/4/2020 is 13 weeks and 4 days. We round that up to 14 weeks and multiple it by £4 = £56 is the maximum that can be claimed in the 2019-20 tax period